apps/examples/artik_demo/artik_demo.c

```diff
@@ -886,6 +880,13 @@ int send_data_to_artik(int *data)
 
 	return 0;
 
+	/* Release memory and close socket if error */
+error_fd_tls:
```

Comment:
What's the meaning of fd_tls?

Reply:
@sunghan-chang 
When 'webclient_tls_init', 'wget_tls_handshake' fail, each function calls 'wget_tls_release', 'wget_tls_ssl_release'.

Reply:
fd_tls meant error related to fd of socket / error related to tls handshake/tls_init